packages/components/src/border-box-control/stories/index.story.tsx

```diff
@@ -16,7 +16,7 @@ import Button from '../../button';
 import { BorderBoxControl } from '../';
 
 const meta: Meta< typeof BorderBoxControl > = {
-	title: 'Components (Experimental)/BorderBoxControl',
+	title: 'Components§/BorderBoxControl',
```

Comment:
```suggestion
	title: 'Components/BorderBoxControl',
```

Typo?

Edit: It appears that this also causes the component styles in the Storybook to break.

Reply:
Oops! Good catch!